pkg/querier/queryrange/limits.go

```diff
@@ -461,9 +485,12 @@ func (rt limitedRoundTripper) Do(c context.Context, request queryrangebase.Reque
 		cancel()
 	}()
 
-	if span := opentracing.SpanFromContext(ctx); span != nil {
+	span := opentracing.SpanFromContext(ctx)
+
+	if span != nil {
 		request.LogToSpan(span)
 	}
+
```

Comment:
What's the purpose of this change?

Reply:
I'd like to move this "span" out so it can be reused below in the handler